test/fixtures/linter/rules/NoGlobals/NoExportedLibValues.js

```diff
@@ -0,0 +1,12 @@
+sap.ui.define(["sap/ui/unified/library", "sap/ui/core/library"],
+	function (unifiedLibrary, coreLibrary) {
+	"use strict";
+	
```

Comment:
Can you add `var SelectOptionSign = compLibrary.smartfilterbar.SelectOptionSign;` (along with an import for "sap/ui/comp/library") here?

For me, it causes a false-positive, as the module is defined as pseudo module, but actually does not exist and is only exported via the library:
```
Access of module 'sap/ui/comp/smartfilterbar/SelectOptionSign' (compLibrary.smartfilterbar.SelectOptionSign) not exported by library 'sap/ui/comp/library'
```

Reply:
Thanks a lot!
I suspected such cases, but couldn't find one. Now it's fixed